head.html

```diff
@@ -1,6 +1,11 @@
 <meta name="viewport" content="width=device-width, initial-scale=1"/>
 <script src="/scripts/aem.js" type="module"></script>
 <script src="/scripts/scripts.js" type="module"></script>
+<script type="text/javascript">
+    window.gnav_jsonPath = "https://cors-anywhere.herokuapp.com/https://www.esri.com/content/experience-fragments/esri-sites/en-us/site-settings/global-navigation-config/2022-nav-config.25.json";
+</script>
+<script src="https://webapps-cdn-dev.esri.com/CDN/components/global-nav/js/gn.js"></script> 
```

Comment:
This doesn't load for me, can you allow it for everyone? If that's an issue, let's discuss alternatives later today.